### PR TITLE
Re to regex

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/utils/parser.py [hunk 1]
```diff
@@ -148,17 +148,17 @@ def chunk(str, chunk_size):
...
         else:
             return value
     elif filter_key == 'remove_first_word':
-        words = re.split(r"([\W]+)", value.strip(), flags=re.UNICODE)
+        words = regex.split(r"([\W]+)", value.strip(), flags=regex.UNICODE)
         if len(words) > 2:
             return "".join(words[2:])
         else:
-            return None
+            return ''
```
Comment (rowanseymour) [inline temba/utils/parser.py]: `re` doesn't mind if the substitution callable returns `None` and converts it to '', but `regex` throws error.

**Comment by nicpottier** [comment]
So from https://pypi.python.org/pypi/regex I gather there are two different modes regex can work in. Are we being explicit about what version is being used? (this bit worries me "If no version is specified, the regex module will default to regex.DEFAULT_VERSION. In the short term this will be VERSION0, but in the longer term it will be VERSION1.")

**Comment by rowanseymour** [comment]
Haven't the Python authors learnt anything about breaking existing code? The way to be explicit is to include "VERSIONX" as a flag with every regex... which sucks a bit... but I suppose it might save us some pain one day.

**Comment by nicpottier** [comment]
Yep.

**Pull request merged**